invenio/modules/search/api.py

```diff
@@ -47,6 +48,7 @@ def parser(self):
         return query_parser
 
     @cached_property
+    @enhance_query()
```

Comment:
@marioskogias please move it inside `search` method as the query enhancers might need access to it's keyword arguments.